### DISCUSSION TITLE
Settings cleanup/redesign

### DISCUSSION THREAD

**Discussion by gkeep** [wstyres/Zebra#1556] [Ideas]
Right now, Settings are _very_ messy. They are unorganized, and for some time now, new options were just dumped there.

I suggest moving 'Featured Packages', 'Changes' and 'Swipe Actions Display As' options under Display section. 
Current 'Display' (accent color, light/dark mode and pure black mode) section renamed to Appearance.

Then, move 'Automatic Refresh', 'Always Install Latest', 'Live Search' and 'Finish Automatically' options under a *new __Behavior__ section*.

All of which would result in a structure like this: 
![image](https://user-images.githubusercontent.com/20600053/93211961-9fee2580-f76a-11ea-8945-c056943dcc4d.png)

**Comment by wstyres** [comment]
A settings redesign is been a to-do for awhile now, we're going to finally do it for 1.2 so it will be addressed in a future beta.
